snowflake/objects/databases/recover/schemas/parquet/tables/enrolledparticipants_customfields_symptoms_parquet.sql

```diff
@@ -0,0 +1,11 @@
+CREATE OR ALTER TABLE ENROLLEDPARTICIPANTS_CUSTOMFIELDS_SYMPTOMS (
```

Comment:
This is neat, does it automatically put you in the right schema?

Reply:
If you are deploying from an entrypoint in `database/recover/schema/parquet/deploy.sql` (where you see the `USE SCHEMA` statement) or higher, than yes. But if you are using another schema and you `EXECUTE IMMEDIATE FROM` this file, then this table will be created in that other schema.